### PR TITLE
Automated cherry pick of #2146: Update GO_BUILD_VERSION

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2146 on release-v1.28.

#2146: Update GO_BUILD_VERSION

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Pull request merged**
